colorbleed/plugins/maya/publish/extract_yeti_rig.py

```diff
@@ -96,9 +100,11 @@ def process(self, instance):
         attr_value = {"%s.imageSearchPath" % n: image_search_path for
                       n in yeti_nodes}
 
-        # get input_SET members
+        # Get input_SET members
         input_set = [i for i in instance if i == "input_SET"]
         members = cmds.sets(input_set[0], query=True)
+        # Get all items
+        members = cmds.listRelatives(members, ad=True, fullPath=True)
```

Comment:
Actually, are you sure this does exactly what you want? :) Because the contents of the set itself are now skipped, only children are remaining in the data.

If you want to have the contents of the set *and* the children you'd need to somehow merge it with whatever  you passed to `cmds.listRelatives` - preferably in a way that would ensure uniqueness; note that `cmds.sets(query=True)` doesn't enforce long paths.